### DISCUSSION TITLE
Is there a way to view what's causing an error.

### DISCUSSION THREAD

**Discussion by adamsanclemente** [Budibase/budibase#4178] [Help]
I'm trying to save a row to a mySQL db using a form and a button. However whenever I click the button it said error saving this row and then nothing happens. Is there a log or something so I can view what's going wrong?

**Comment by mjashanks** [comment]
Try opening your browsers dev tools and looking at the network tab.

You will see a failed request in red, after you click save. 

View the response - this might give you a better error message. Not ideal, sorry - we should do better at this.

**Reply by zoggo19** [reply]
Hey @PixelParallax , sorrry i had to tag you. I was wondering, did you connect budibase directly with mysql? Because, i have been trying to connect it with MySQL using the configurations as per MySQL workbench and it keeps failing to connect. Are there alternatives or tricks i could use to overcome this hurdle?
![hurdle 1](https://user-images.githubusercontent.com/55872718/192360351-9ccad40a-12ef-4766-9f38-42dd7f5329c8.PNG)

**Reply by adamsanclemente** [reply]
@zoggo19 It should be pretty simple, just whitelist the ip of budibase in your MySQL host, then assuming the info is correct it should just work.

**Reply by zoggo19** [reply]
I already did that
![hurdle 2](https://user-images.githubusercontent.com/55872718/192362863-2cb124cf-69b4-4590-8bb9-ca68ccf683e5.PNG)

**Reply by adamsanclemente** [reply]
Welp, I don't know what to tell you. Fortunately for you Budibase has an active discord now which you can ask in [https://discord.gg/K3CPPaHGz5](https://discord.gg/K3CPPaHGz5)

**Reply by zoggo19** [reply]
Are you using docker?

**Reply by adamsanclemente** [reply]
I was using the budibase provided hosting at the time. I no longer have a use for budi base

**Reply by zoggo19** [reply]
Oh, okay then. Thank you alot.
